GUIController.py

```diff
@@ -81,14 +81,16 @@ def addRelationship(self, firstClassName, secondClassName, typ):
     def deleteRelationship(self, firstClassName, secondClassName):
         try:
             self.model.deleteRelationship(firstClassName, secondClassName)
+            self.view.deleteLine(firstClassName, secondClassName)
         except Exception as e:
             print(e)
 
         print(self.model.relationshipDict)
 
-    def renameRelationship(self, firstClassName, secondClassName, typ):
+    def changeRelationship(self, firstClassName, secondClassName, typ):
```

Comment:
If the method name is changed to this, the method called in the change relationship menu needs to be changed too, or it breaks things